install/kubernetes/addons/prometheus.yaml

```diff
@@ -28,7 +28,7 @@ metadata:
 spec:
   selector:
     app: prometheus
-  type: LoadBalancer
```

Comment:
Yes. Related to https://github.com/istio/istio/issues/255. While I was playing with prometheus, I thought we should correct this in this PR.